app/src/main/javascript/bundles/gene-search/src/ExperimentCard.js

```diff
@@ -54,7 +54,7 @@ class ExperimentCard extends React.Component {
            {experimentDescription}
             <SmallIconDiv>
               {showIcon && <ExperimentIconDiv background={`indianred`} color={`white`} data-toggle={`tooltip`} data-placement={`bottom`}
-                                              title={`Experiment with annotated data analysed by an external source.`}>A</ExperimentIconDiv>}
+                                              title={`Externally analysed data.`}>E</ExperimentIconDiv>}
```

Comment:
Tiny thing: I don't think we need the `.`. If you check tooltips for example in GitHub, none of them have `.` at the end. Could you remove it, please?
Thanks